hack/sync-api.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function usage() {
+    cat <<EOF
+Build components:
+    $0 <install-filter-repo|filter-repo|sync-api>
+Build plugins:
+    $0 plugins <filter-repo>
+ENV:
+    API_ROOT: clusterpedia api path, required if the pwd is not the api path.
+        eg. API_ROOT=/home/runner/work/clusterpedia/staging/src/github.com/clusterpedia-io/api
+    API_REPO: API_REPO: API_REPO git url with github token.
+        eg. API_REPO=https://$GH_TOKEN@github.com/clusterpedia/api.git
+EOF
+}
+
+set +e; REPO_ROOT=$(git rev-parse --show-toplevel 2>/dev/null);set -e
+if [ -z $REPO_ROOT ]; then
+    if [ -z $API_ROOT ]; then
+        echo "the current directory is not in the clusterpedia api path, please set API_ROOT=<clusterpedia api path>"
+        usage
+    exit 1
+    fi
+        REPO_ROOT=$(pwd)
+else
+    API_ROOT="${REPO_ROOT}/staging/src/github.com/clusterpedia-io/api"
+fi
+
+if [ -z $GH_TOKEN ]; then
+    if [ -z $API_REPO ]; then
+        echo "the github token is not in the env, please check CLUSTERPEDIA_BOT_TOKEN"
+        usage
+    exit 1
+    fi
+else
+    API_REPO="https://$GH_TOKEN@github.com/clusterpedia/api.git"
+fi
+
+raw=$(git branch -r --contains $REF)
+BRANCH_NAME=${raw/origin\/}
+
+
+install_filter_repo(){
+    python3 -m pip install --user git-filter-repo
+}
+
+TAG_MESSAGE=""
+# 获取 clusterpedia 的 tag message
+init_tag_message(){
```

Comment:
This seems unnecessary to wrap as a function

Reply:
thanks i have fixed it